package.json

```diff
@@ -9,11 +9,12 @@
   "scripts": {
     "apidocs": "./node_modules/.bin/jsdoc2md",
     "browser-tests": "NODE_ENV=test ./node_modules/.bin/karma start --browsers Chrome",
-    "build": "NODE_ENV=production npm test && npm run clean && ./node_modules/.bin/webpack --config package.config.js --optimize-minimize",
+    "build": "NODE_ENV=production npm run clean && ./node_modules/.bin/webpack --config package.config.js --optimize-minimize",
     "build-docs": "./update-gh-pages.sh",
     "clean": "./node_modules/.bin/rimraf ./dist/*",
     "lint": "./node_modules/.bin/eslint .storybook/ src/ stories/ test/ *.js",
-    "prepublish": "npm run build",
+    "prepare": "npm run build",
```

Comment:
I think it's probably alright to have it the way it is here. It is a little awkward, but it only happens on `publish`, which is not terribly often and it does solve the issue with having it test and lint on install.